python/nano/src/bigdl/nano/deps/openvino/core/openvino_model.py

```diff
@@ -14,8 +14,13 @@
 # limitations under the License.
 #
 from pathlib import Path
+from tempfile import TemporaryDirectory
 from openvino.runtime import Core
 from openvino.runtime.passes import Manager
+from openvino.tools.pot.graph import load_model, save_model
```

Comment:
Instead of unconditionally importing pot, make it an optional dependency (i.e., users only need to install pot if they use it).

Reply:
POT is installed with `openvino-dev`. So we don't have to check if pot is installed as long as openvino is installed.